### PR TITLE
feat: add search in govt website list page

### LINKED ISSUE #1: Add search in government website list page
## What

Since the list can be long and finding a specific government website can be hard, we will need to add a search bar in the goverment websites page

## How

The package fuse.js has been already added, need to implement search using this package.

> [!IMPORTANT]
> "threshold" should be set to 0.3
> "ignoreLocation" should be set to true

### PR DESCRIPTION
**Pull request by susanhsrestha**
The search functionality for government websites was added, as mentioned on #1 
Closes #1

### CODE REVIEW

**Review by bsantosh909 — CHANGES_REQUESTED** [review]
Thanks for the PR, however since the entire website is on dark theme, I believe having the search input in dark will also be more aesthetic.

Maybe you can take reference from the other PR made for similari issue and update your code?
https://github.com/Know-Nepal/website/pull/5

**Comment by susanhsrestha** [comment]
Sure, I'll look into it

File: src/pages/government-websites.tsx [hunk 1]
```diff
@@ -15,15 +16,38 @@ export default function GovernmentWebsitesPage() {
     initialData: [],
   });
 
-  //
+  const [searchQuery, setSearchQuery] = useState("");
+  const fuseOptions = {
+    keys: ["name", "description"],
```
Comment (thenishantsapkota) [inline src/pages/government-websites.tsx]: It would be better if the search key was `name` only!

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: src/pages/government-websites.tsx (hunk 1 above)
Comment (susanhsrestha) [reply src/pages/government-websites.tsx]: Done! The search key has been updated to 'name' only. Thank you!

**Review by thenishantsapkota — DISMISSED** [review]
LGTM!

File: src/pages/government-websites.tsx [hunk 2]
```diff
@@ -1,29 +1,53 @@
...
   return (
     <div>
-      <p className="text-center text-3xl">Government Websites of Nepal</p>
-
-      {isFetching && <p>Loading data...</p>}
+      <p className="text-center text-3xl">Government Websites in Nepal</p>
+      <div className="my-10 flex justify-center">
+        <input
+          type="text"
+          placeholder="Search for govenment website..."
```
Comment (thenishantsapkota) [inline src/pages/government-websites.tsx]:
```suggestion
          placeholder="Search for government website..."
```

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Review by bsantosh909 — APPROVED** [review]
LGTM, thank you for your contribution

**Pull request merged**